File: Python/6-Kyu/Multiples-of-3-or-5.py
# Multiples of 3 or 5
def solution(number):
    
    sum = 0

    for i in range(0, number):
        if i % 3 == 0 or i % 5 == 0:
            sum += i
    
    return(sum)

# Test
print(solution(10))  # --> 23
print(solution(15))  # --> 45

File: Javascript/6-Kyu/Multiples-of-3-or-5.js
// Multiples of 3 or 5
function solution(number){

    let sum = 0;

    for ( let num = 0; num < number; num++ ) {
        if ( num % 3 === 0 || num % 5 === 0) {
            sum += num
        }
    };

    return sum
  
};

// Test
console.log(solution(10))  // --> 23
console.log(solution(15))  // --> 45
